srcpkgs/texlive/template

```diff
@@ -0,0 +1,346 @@
+# Template file for 'texlive'
+pkgname=texlive
+version=20200406
+revision=1
+wrksrc="texlive-${version}-source"
+build_wrksrc="build"
+build_style=gnu-configure
+configure_script="../configure"
+configure_args="
+ --enable-biber
+ --enable-epsfwin
+ --enable-ipc
+ --enable-luajithbtex
+ --enable-luatex
+ --enable-mftalkwin
+ --enable-regiswin
+ --enable-shared
+ --enable-tektronixwin
+ --enable-unitermwin
+ --enable-xetex
+ --enable-dvipng
+ --enable-dvipsk
+ --enable-ps2eps
+ --enable-psutils
+ --disable-bibtex-x
+ --disable-chktex
+ --disable-cjkutils
+ --disable-detex
+ --disable-dialog
+ --disable-dvi2tty
+ --disable-dvisvgm
+ --disable-largefile
+ --disable-lcdf-typetools
+ --disable-multiplatform
+ --disable-native-texlive-build
+ --disable-pdfopen
+ --disable-ps2pkm
+ --disable-t1utils
+ --disable-tex4htk
+ --disable-ttf2pk2
+ --disable-vlna
+ --disable-xindy
+ --with-ps=gs
+ --with-banner-add=/VoidLinux
+ --with-system-cairo
+ --with-system-freetype2
+ --with-system-gd
+ --with-system-graphite2
+ --with-system-gmp
+ --with-system-harfbuzz
+ --with-system-icu
+ --with-system-libpaper
+ --with-system-libpng
+ --with-system-mpfr
+ --with-system-poppler
+ --with-system-pixman
+ --with-system-teckit
+ --with-system-xpdf
+ --with-system-zlib
+ --with-system-zziplib
+ --with-xdvi-x-toolkit=Xaw"
+hostmakedepends="pkg-config perl lua52-BitOp texinfo"
+makedepends="cairo-devel freetype-devel gd-devel graphite-devel gmp-devel
+ harfbuzz-devel icu-devel libpaper-devel libpng-devel mpfr-devel
+ poppler-devel pixman-devel libteckit-devel zlib-devel zziplib-devel
+ libXaw-devel"
+depends="dialog ghostscript openjdk-jre perl-Tk"
```

Comment:
Should depend on texlive-core

Reply:
I'm still not entirely convinced on that. The texmf distribution seems to be somewhat more isolated from the texlive binaries than I previously thought. If someone developed an alternative texmf distribution, then there would be two texmf options, and you could use either... I was thinking to just depend on texlive-core in each of the metapackages.

Feel free to correct me, I'm not sure my reasoning is very logical, and since there is only really one texmf distribution now, maybe your suggestion makes sense.

Reply:
nobody's gonna develop another distribution, and the texlive package literally needs some perl scripts from texlive-core to work at all